src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -98,6 +101,13 @@ internal PackageSourceMoniker SelectedSource
         private readonly Guid _sessionGuid = Guid.NewGuid();
         private readonly Stopwatch _sinceLastRefresh;
 
+        // Helper class to pass back multiple package feeds
+        private class PackageFeeds
```

Comment:
You can use a value tuple instead of a class:
https://docs.microsoft.com/en-us/dotnet/api/system.valuetuple?view=netframework-4.8

Reply:
Fixed